### ISSUE TITLE
Timer display icon is not working correctly on Google Chrome

### ISSUE THREAD

**Issue by bexeru** [schmich/marinara#509]
![Captura de pantalla 2023-11-11 a la(s) 4 10 52 p m](https://github.com/schmich/marinara/assets/150557334/1086063d-9164-4f3f-9ca6-860a39ed95bb)

The Icon display should be 25 instead of 5. Timer went crazy I can't set it up correctly. I've already uninstalled the extension and reinstalled it and keeps behaving the same. 
I hope someone fix it. 

This is happening since yesterday after Chrome Updated. 
I'm currently working on IOS MAC MINI M1

**Comment by dianaxtest** [comment]
Hello! Timer is not actually "crazy" since the counter is working properly. Ex:

![2023-11-12_12h04_11](https://github.com/schmich/marinara/assets/150608670/c86f4275-d4c2-4dd5-8df5-330ca028ce9f)

Even though it show just a 4, when going over the extension icon with your cursor, you'll be able to see the correct time. It's actually just a matter of display since the new google chrome update. It'll have to be taken in consideration for future pomodoro extension fixes.

**Comment by alexol26** [comment]
I have the same issue. The icon doesn't work properly

**Comment by AndBerna** [comment]
Same here :(

**Comment by dimateos** [comment]
Same here, in my case I can see a few pixels of the first number:
![image](https://github.com/schmich/marinara/assets/26868611/9e43ba03-00a5-4493-bbee-b14ed7dd0e34)

* **Potential solution**: drop de "min" suffix (it is always mins). Other option is to reduce it to "m" or "'", maybe simply have a setting for the suffix and delegate to the user the decision.
  * For timing under a minute (I dont remember what it used to display), but it cuild be <1 or 0 both comprehensive enough.
  
Best regards!

**Comment by alexol26** [comment]
> * **Potential solution**: drop de "min" suffix (it is always mins). Other option is to reduce it to "m" or "'", maybe simply have a setting for the suffix and delegate to the user the decision.

I agree. The simplest way to fix it is to drop 'min' or change to 'm'.

**Comment by alandromundo** [comment]
Same issue here. Can't see the full timer at the moment
![image](https://github.com/schmich/marinara/assets/66541404/4e18f883-9e13-4b86-8eaa-126680ee3aa2)

I'm working on Chrome Dev version 128.0.6535.2
